### ISSUE TITLE
License Choice

### ISSUE THREAD

**Issue by rita-ni** [UBC-MDS/imgtoolPy#56]
Here we will discuss the rationale for our license choice for our project

**Comment by franklu2014** [comment]
I think we should first rule out Creative Commons license because we are not producing a new image

**Comment by rita-ni** [comment]
> I think we should first rule out Creative Commons license because we are not producing a new image

I agree, we are using copyright-free images as examples are not intended for commercial creation of images. Therefore I think MIT license would suffice for our project.

**Comment by Margaret8521** [comment]
I agree as well since we want others to contribute to our package, we need a license that is widespread with unrestricted usage and MIT fits our needs.

**Comment by franklu2014** [comment]
Agreed.  Based on the nature of our project, we should use the MIT license.
